src/ripple/rpc/handlers/ServerInfo.cpp

```diff
@@ -22,13 +22,298 @@
+            a[0U] = "taker_gets_funded";
+            Json::Value v = Json::objectValue;
+            v[jss::nth] = 258;
+            v[jss::isVLEncoded] = false;
+            v[jss::isSerialized] = false;
+            v[jss::isSigningField] = false;
+            v[jss::type] = "Amount";
+            a[1U] = v;
+            ret[jss::FIELDS][i++] = a;
+        }
+
+        {
+            Json::Value a = Json::arrayValue;
+            a[0U] = "taker_pays_funded";
+            Json::Value v = Json::objectValue;
+            v[jss::nth] = 259;
+            v[jss::isVLEncoded] = false;
+            v[jss::isSerialized] = false;
+            v[jss::isSigningField] = false;
+            v[jss::type] = "Amount";
+            a[1U] = v;
+            ret[jss::FIELDS][i++] = a;
+        }
+
+        for (auto const& [code, f] : ripple::SField::knownCodeToField)
+        {
+            if (f->fieldName == "")
+                continue;
+
+            Json::Value innerObj = Json::objectValue;
+
+            uint32_t type = f->fieldType;
+
+            innerObj[jss::nth] = f->fieldValue;
+
+            innerObj[jss::isVLEncoded] =
+                (type == 7U /* Blob       */ || type == 8U /* AccountID  */ ||
+                 type == 19U /* Vector256  */);
+
+            innerObj[jss::isSerialized] =
+                (type < 10000 && f->fieldName != "hash" &&
+                 f->fieldName !=
+                     "index"); /* hash, index, TRANSACTION, LEDGER_ENTRY,
+                                  VALIDATION, METADATA */
+
+            innerObj[jss::isSigningField] = f->shouldInclude(false);
+
+            innerObj[jss::type] = type_map[type];
+
+            Json::Value innerArray = Json::arrayValue;
+            innerArray[0U] = f->fieldName;
+            innerArray[1U] = innerObj;
+
+            ret[jss::FIELDS][i++] = innerArray;
+        }
+
+        ret[jss::TRANSACTION_RESULTS] = Json::objectValue;
+
+        for (const auto& pair : detail::transResults())
+        {
+            ret[jss::TRANSACTION_RESULTS][pair.second.first] = pair.first;
+        }
+
+        ret[jss::TRANSACTION_TYPES] = Json::objectValue;
+        ret[jss::TRANSACTION_TYPES][jss::Invalid] = -1;
+        for (auto it = TxFormats::getInstance().begin();
+             it != TxFormats::getInstance().end();
+             ++it)
+        {
+            ret[jss::TRANSACTION_TYPES][it->getName()] = it->getType();
+        }
+
+        // generate hash
+        {
+            const std::string out = Json::FastWriter().write(ret);
+            defsHash_ =
+                ripple::sha512Half(ripple::Slice{out.data(), out.size()});
+            ret[jss::hash] = to_string(defsHash_);
+        }
+
+        return ret;
+    }
+
+    uint256 defsHash_;
+    Json::Value defs_;
+
+public:
+    Definitions() : defs_(generate()){};
+
+    bool
+    hashMatches(uint256 hash) const
+    {
+        return defsHash_ == hash;
+    }
+
+    Json::Value const&
+    operator()() const
```

Comment:
This doesn't really need to be an operation. I'd probably make this a regular function. But also fine as-is if there's a reason I'm missing or you strongly prefer the operator.

Reply:
Fixed in https://github.com/XRPLF/rippled/pull/4703/commits/7f17a784dccd4a5ac232470275f2dc52cb7f6c45